resources/public/api/conf/5.0/foreign_property_cumulative_summary_retrieve.yaml

```diff
@@ -0,0 +1,99 @@
+
+  Note: This endpoint only supports submissions for tax year 2025-26 onwards.
+
+  ### Test data
+  Scenario simulation using Gov-Test-Scenario headers is only available in the sandbox environment.
+  
+  | Header Value (Gov-Test-Scenario) | Scenario                                                                 |
+  |----------------------------------|--------------------------------------------------------------------------|
+  | N/A - DEFAULT                    | Simulates success response with Foreign Property.                        |
+  | NOT_FOUND                        | Simulates the scenario where no data is found.                           |
+  | FOREIGN_PROPERTY_FULL_EXPENSES   | Simulates success response of a Foreign property with full expenses.     |
+  | FOREIGN_PROPERTY_CONSOLIDATED    | Simulates the scenario of a Foreign property with consolidated expenses. |
+  | UK_PROPERTY                      | Simulates the scenario where the type of business is incorrect.          |
+  | STATEFUL                         | Performs a stateful retrieve.                                            |
+
+
+tags:
+  - Foreign Property Cumulative Period Summary
+
+parameters:
+  - $ref: './common/pathParameters.yaml#/components/parameters/nino'
+  - $ref: './common/pathParameters.yaml#/components/parameters/businessId'
+  - $ref: './common/pathParameters.yaml#/components/parameters/taxYearFrom2025'
+  - $ref: './common/pathParameters.yaml#/components/parameters/submissionId'
+  - $ref: './common/headers.yaml#/components/parameters/acceptHeader'
+  - $ref: './common/headers.yaml#/components/parameters/authorizationReadScopeHeader'
+  - $ref: './common/headers.yaml#/components/parameters/testScenarioHeader'
+
+security:
+  - User-Restricted:
+      - read:self-assessment
+
+responses:
+  "200":
+    description: Success
+    headers:
+      X-CorrelationId:
+        $ref: './common/headers.yaml#/components/parameters/correlationId'
+      Deprecation:
+        $ref: './common/headers.yaml#/components/parameters/deprecationHeader'
+      Sunset:
+        $ref: './common/headers.yaml#/components/parameters/sunsetHeader'
+      Link:
+        $ref: './common/headers.yaml#/components/parameters/linkHeader'
+    content:
+      application/json:
+        schema:
+          $ref: './schemas/foreign_property_cumulative_summary_retrieve/def1/response.json'
+        examples:
+          Full expenses response for TY 2025-26 and after:
+            value:
+              $ref: './examples/foreign_property_cumulative_summary_retrieve/def1/response.json'
+          Consolidated expenses response for TY 2025-26 and after:
+            value:
+              $ref: './examples/foreign_property_cumulative_summary_retrieve/def1/consolidated_response.json'
+
+  "400":
+    description: Bad request
+    content:
+      application/json:
+        schema:
+          $ref: './schemas/error-response.yaml'
```

Comment:
It seems this doesn't align with what we currently have in our codebase. At the moment, it lists code as required and error message as optional. However, in our codebase, both the code and message fields are required, while paths is optional. This can be verified [here](https://github.com/hmrc/property-business-api/blob/main/app/api/models/errors/MtdError.scala#L22) and [here](https://github.com/hmrc/property-business-api/blob/main/app/api/models/errors/MtdError.scala#L55-L59).

This discrepancy affects not only this API, so it may be something to consider and rectify across all APIs.

In the future, the [error-response](https://github.com/hmrc/property-business-api/blob/main/resources/public/api/conf/5.0/schemas/error-response.yaml) may be updated to something like the following:

```
description: Error Response
type: object
properties:
  code:
    description: The error code
    type: string
  message:
    description: The error message
    type: string
  paths:
    type: array
    description: The paths to the specific nodes where the errors occur
    items:
      type: string
required:
  - code
  - message
```